src/main/java/org/sonarlint/intellij/ui/tree/SecurityHotspotTreeModelBuilder.java

```diff
@@ -63,8 +63,9 @@ public class SecurityHotspotTreeModelBuilder implements FindingTreeModelBuilder
   private static final Comparator<LiveSecurityHotspot> SECURITY_HOTSPOT_COMPARATOR = new SecurityHotspotComparator();
   private static final Comparator<LiveSecurityHotspotNode> SECURITY_HOTSPOT_WITHOUT_FILE_COMPARATOR = new LiveSecurityHotspotNodeComparator();
 
+  protected SecurityHotspotFilters currentFilter = SecurityHotspotFilters.DEFAULT_FILTER;
+  protected boolean currentResolvedFilter = false;
```

Comment:
I would suggest to improve the name `currentResolvedFilter`,it is misleading as it is not a filter. Something like `shouldShowResolvedHotspots` ?

Reply:
Good point :+1: